mex/editor/state.py

```diff
@@ -86,6 +88,17 @@ def _update_raw_path(
             raw_path = f"{raw_path}?{query_str}"
         nav_item.raw_path = raw_path
 
+    @rx.event
+    def change_locale(self, locale: str) -> EventSpec:
+        """Change the current locale to the given one and reload the page.
+
+        Args:
+            locale: The locale to change to.
+        """
+        LocaleService.get().set_locale(locale)
+        self.current_locale = locale
+        return rx.call_script("window.location.reload()")
```

Comment:
I am uncertain about the implications of this. Do we loose all state if we trigger a reload? If yes: Should we rather do this dynamically? @cutoffthetop What do you think?